tests/h/views/badge_test.py

```diff
@@ -14,15 +14,22 @@ class TestBlocklist:
     @pytest.mark.parametrize("domain", Blocklist.BLOCKED_DOMAINS)
     @pytest.mark.parametrize("prefix", ("http://", "https://", "httpx://", "//"))
     @pytest.mark.parametrize("suffix", ("", "/", "/path?a=b"))
-    def test_it_blocks(self, domain, prefix, suffix):
+    def test_it_blocks_bad_domains(self, domain, prefix, suffix):
         assert Blocklist.is_blocked(f"{prefix}{domain}{suffix}")
 
+    @pytest.mark.parametrize("scheme", Blocklist.BLOCKED_SCHEMES)
+    @pytest.mark.parametrize("suffix", ("://about", "://newtab"))
+    def test_it_blocks_bad_schema(self, scheme, suffix):
+        assert Blocklist.is_blocked(f"{scheme}://{suffix}")
+
     @pytest.mark.parametrize(
         "acceptable_url",
         (
             "http://example.com/this/is/fine",
             "http://example.com//facebook.com",
             "http://facebook.com.om.nom",
+            "file://c/my/magical_file.pdf",
+            "chrome-extension://blah",
```

Comment:
Add some edge cases to make sure:

 * We aren't blocking everything that isn't `http*`
 * That we aren't blocking anything that just starts with `chrome`